src/views/Home/index.tsx

```diff
@@ -85,7 +85,7 @@ const Home = ({ history }: HomeProps) => {
             <Button
               type="button"
               onClick={() => {
-                history.push('/system/new');
+                history.push('/governance-overview');
```

Comment:
🎨 If you have time, it might be worth refactoring this button to be a `Link`, too.